app/invocation/invocation_cache_card.tsx

```diff
@@ -25,8 +27,8 @@ export default class CacheCardComponent extends React.Component {
           {this.props.model.cacheStats.length && (
             <div className="details">
               {this.props.model.cacheStats.map((cacheStat) => {
-                let downloadThroughput = +cacheStat.downloadThroughputBytesPerSecond / 1000000 || 0;
-                let uploadThroughput = +cacheStat.uploadThroughputBytesPerSecond / 1000000 || 0;
+                let downloadThroughput = bitsPerByte * (+cacheStat.downloadThroughputBytesPerSecond / 1000000) || 0;
```

Comment:
nit: `|| 0` can be removed since all types in the expression are `number`

Reply:
Done